config/install/image.style.bg_rich_text_narrow.yml

```diff
@@ -13,3 +13,4 @@ effects:
     data:
       width: '740'
       height: '663'
+      crop_type: focal_point
```

Comment:
@Saphyel why do you have all those unrelated commits in this PR - this doesn't look right?